packages/graphql-migrations/src/util/parseAnnotations.ts

```diff
@@ -0,0 +1,14 @@
+import { parseMetadata } from 'graphql-metadata'
+
+/**
+ * Wraps `parseMetadata` to return empty object instead of undefined
+ * As lots of graphql-migration uses this.
+ * 
+ * @param namespace
+ * @param description
+ */
+export function parseAnnotations(namespace: string, description: string): any {
```

Comment:
A good candidate for the core, we already have https://github.com/aerogear/graphback/blob/master/packages/graphback-core/src/annotations/parser.ts#L4

Reply:
Actually I will remove `parseDbAnnotations`. I added this in `graphql-migrations` because the library heavily relies on the assumption that an empty object will be returned for `@db` so it does not do any checking for undefined. `parseMetadata` returns `true` so these approaches are incompatible. This function wraps `parseMetadata` to ensure the output stays the same in graphql-migrations. I will change the name to make it clear.